android/src/main/java/com/pusherman/networkinfo/RNNetworkInfo.java

```diff
@@ -21,20 +21,18 @@
 import java.util.Arrays;
 import java.util.List;
```

Comment:
I had a test of this on a Pixel 2 (API 29) emulator and got this error when running

```

> Task :react-native-network-info:compileDebugJavaWithJavac FAILED
app\node_modules\react-native-network-info\android\src\main\java\com\pusherman\networkinfo\RNNetworkInfo.java:179: error: cannot find symbol
                            if (addr instanceof Inet6Address)
                                                ^
  symbol: class Inet6Address
1 error

FAILURE: Build failed with an exception.
```

I got it fixed by adding this import

```suggestion
import java.util.List;
import java.net.Inet6Address;
```

Reply:
Thank you for catching that Chris! I've committed your change to the PR.